privval/socket.go

```diff
@@ -35,51 +36,25 @@ var (
 
 var (
 	acceptDeadline = time.Second * defaultAcceptDeadlineSeconds
-	connDeadline   = time.Second * defaultConnDeadlineSeconds
+	connTimeout    = time.Second * defaultConnDeadlineSeconds
 	connHeartbeat  = time.Second * defaultConnHeartBeatSeconds
 )
 
-// SocketPVOption sets an optional parameter on the SocketPV.
-type SocketPVOption func(*SocketPV)
```

Comment:
Why were all the options removed? They provide a way to tune vital aspects of the implementation from outside the package. Reasoning here is appreciated, but ideally those deletions should be reverted.

Reply:
I thought they were unnecessary to add becuase nonetheless these options can right now not be modified after OnStart is called. But outside modification makes sense so I will reintroduce them.

@xla do you think we should refactor the implementation to allow modifications after Start is called ?